shinken/objects/item.py

```diff
@@ -205,8 +205,14 @@ def clean(self):
             except AttributeError:
                 pass
 
+    def get_name(self):
+        return getattr(self, 'name', "unknown")
+
     def __str__(self):
-        return str(self.__dict__) + '\n'
+        cls_name = self.__class__.__name__
+        return '<%s "name"=%r />' % (cls_name, self.get_name())
+
+    __repr__ = __str__
```

Comment:
this simply define `__repr__` as an alias for `__str__`, without having to do something like:
```
def __repr__(self):
    return self.__str__()  # or better : return str(self)
```